src/analysis/static/graph/drawGraph.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Do these need to be above the copyright?

Reply:
Nope, fixed